components/notifications.js

```diff
@@ -283,7 +283,7 @@ function Invitification ({ n }) {
     <>
       <NoteHeader color='secondary'>
         your invite has been redeemed by
-        {numWithUnits(n.invite.invitees.length, {
+        {' ' + numWithUnits(n.invite.invitees.length, {
```

Comment:
this was a bug
before:
`your invite has been redeemed by1 stacker`
now:
`your invite has been redeemed by 1 stacker`